### ISSUE TITLE
npm broken for ionic 1.2 series

### ISSUE THREAD

**Issue by AgDude** [driftyco/ionic-cli#1028]
```
npm install ionic@1.2.13
```
from npm-debug.log:
```
20316 http fetch 504 https://registry.npmjs.org/i/-/i-0.3.2.tgz
20317 info retry will retry, error on last attempt: Error: server error 504
```
Here is the explanation: https://github.com/pksunkara/inflect/issues/11

I have also opened a post on the forum: https://forum.ionicframework.com/t/npm-504-error-on-installing-ionic-1-2/53552

**Comment by AgDude** [comment]
This is referenced in the shrinkwrap for "prompt".

**Comment by jgw96** [comment]
Hello all! Is this still an issue with the latest version of ionic? Thanks!

**Comment by jgw96** [comment]
Hello all! As it seems it has been a while since there was any activity on this issue i will be closing it for now. Feel free to comment if you are still running into this issue. Thanks for using Ionic!

**Comment by AgDude** [comment]
This is an issue with the 1.2 series.

**Comment by jgw96** [comment]
Hello @AgDude! Does it work with ionic 1.3.1?

**Comment by chatterburger97** [comment]
@jgw96  I am trying to install ionic v1.3.1, and I ran into the exact same issue. I ran 
`npm install -g ionic`
and the resulting output was

`npm ERR! fetch failed https://registry.npmjs.org/i/-/i-0.3.2.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 504
npm ERR! fetch failed https://registry.npmjs.org/i/-/i-0.3.2.tgz
npm WARN retry will retry, error on last attempt: Error: fetch failed with status code 504
[    ..............] / loadRequestedDeps: WARN retry will retry, error on last`

I have tried linking directly to the repository as well with 
 `npm install https://registry.npmjs.org/ionic/-/ionic-1.3.1.tgz` but it gives the same output.

I also uninstalled npm and node, and reinstalled them using brew. Still getting the same output. 

I am running into this problem as I accidentally upgraded my Ionic CLI to v2.0.0, but would like to have the old v1.3.1 back.
